layer1/transaction/backend.go

```diff
@@ -166,7 +166,7 @@ func (g *group) sendReceipt(logger *logrus.Entry, receipt *types.Receipt, err er
 // making sure that struct conforms the interface
 var _ ReceiptResponse = &SharedReceipt{}
 
-// Struct to send and share a receipt retrieved by the watcher
+// SharedReceipt is a struct to send and share a receipt retrieved by the watcher
```

Comment:
Doc comments can start with an article (A, An, The) if it makes it a better sounding sentence.

```suggestion
// A SharedReceipt retrieved by the watcher.
```